mitodl/mit-open
# content file weight

### What are the relevant tickets?
closes https://github.com/mitodl/hq/issues/5721

### Description (What does it do?)
This adds a slider to the the admin search section to control whether to include content file data in the search results for searches with a text query

### How can this be tested?
run ./manage.py backpopulate_ocw_data --course-name 6-006-introduction-to-algorithms-spring-2020

http://open.odl.local:8062/search/?q=dirt should return that course 

http://open.odl.local:8062/search/?q=dirt&content_file_score_weight=0  should not return it

Verify that as an admin you see "Content File Score Weight Adjustment" in the admin facet sidebar and can set it

Verify that search works normally otherwise
